src/index.html

```diff
@@ -2,26 +2,447 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
     <title>Catalog</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
     <link
       rel="preconnect"
       href="https://fonts.gstatic.com"
     />
+
     <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
       rel="stylesheet"
     />
+
     <link
       rel="stylesheet"
       href="styles/index.scss"
```

Comment:
Browsers cannot interpret SCSS files directly. You need to compile your SCSS file into a CSS file and link to the resulting CSS file instead. Update the `href` attribute to point to the compiled CSS file.